src/merge/mergeTilesTasker.ts

```diff
@@ -136,7 +136,8 @@ export class MergeTilesTasker {
     grids: Grid[],
     extent: BBox,
     managerCallbackUrl: string,
-    isNew?: boolean
+    isNew?: boolean,
+    previousRelativePath?: string
```

Comment:
I prefer you will create a "cleanup" object with "previousRelativePath" as a variable inside that object - for future expansion